lib/init/passport.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * @fileoverview Passport authentication init
+ */
+const config = require('config');
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
```

Comment:
We're eventually going to want to expose the strategies in a configurable way, but this is OK for now